### PR TITLE
Add vee

### PR DESCRIPTION
**Pull request by artivis**
Add 'vee', the API is shown in the following pseudo-code:

```bash
LieAlg lie_alg = Tangent::Random().hat();
---
Tangent tangent = Tangent::Vee( lie_alg );
// or
tangent.setVee( lie_alg );
```

### CODE REVIEW

File: include/manif/impl/rn/RnTangent_base.h [hunk 1]
```diff
@@ -202,6 +202,7 @@ struct RandomEvaluatorImpl<RnTangentBase<Derived>>
   }
 };
 
+//! @brief Bracket specialization for RnTangentBase objects.
```
Comment (joansola) [inline include/manif/impl/rn/RnTangent_base.h:205]: Does this bracket thing belong to this PR?

**Review by joansola — COMMENTED** [review]
Hey. Just one small comment on a "bracket" thing.

File: include/manif/impl/rn/RnTangent_base.h (hunk 1 above)
Comment (artivis) [reply include/manif/impl/rn/RnTangent_base.h:205]: Yeah it's just a small comment/doc that's missing and which I stuff in this pr.

**Review by joansola — APPROVED** [review]
OK

**Pull request merged**